### ISSUE TITLE
DEBUG    : place_items: invalid item group when generating a hunting lodge basement

### ISSUE THREAD

**Issue by hexagonrecursion** [CleverRaven/Cataclysm-DDA#50647]
### Describe the bug

When I approach a hunting lodge I get a bunch of errors:
```
 DEBUG    : place_items: invalid item group 'bullwhip', om_terrain = 'lodge_basement_residential1_north' (lodge_basement_residential1)

 FUNCTION : std::vector<item*> map::place_items(const item_group_id&, int, const tripoint&, const tripoint&, bool, const time_point&, int, int)
 FILE     : src/mapgen.cpp
 LINE     : 5254
 VERSION  : ed2f630

 DEBUG    : place_items: invalid item group 'straw_doll', om_terrain = 'lodge_basement_residential1_west' (lodge_basement_residential1)

 FUNCTION : std::vector<item*> map::place_items(const item_group_id&, int, const tripoint&, const tripoint&, bool, const time_point&, int, int)
 FILE     : src/mapgen.cpp
 LINE     : 5254
 VERSION  : ed2f630

 DEBUG    : place_items: invalid item group 'bag_body_bag', om_terrain = 'lodge_basement_residential1_west' (lodge_basement_residential1)

 FUNCTION : std::vector<item*> map::place_items(const item_group_id&, int, const tripoint&, const tripoint&, bool, const time_point&, int, int)
 FILE     : src/mapgen.cpp
 LINE     : 5254
 VERSION  : ed2f630

 DEBUG    : place_items: invalid item group 'coke', om_terrain = 'lodge_basement_residential1_west' (lodge_basement_residential1)

 FUNCTION : std::vector<item*> map::place_items(const item_group_id&, int, const tripoint&, const tripoint&, bool, const time_point&, int, int)
 FILE     : src/mapgen.cpp
 LINE     : 5254
 VERSION  : ed2f630


 DEBUG    : place_items: invalid item group 'plastic_bag', om_terrain = 'lodge_basement_residential1_west' (lodge_basement_residential1)

 FUNCTION : std::vector<item*> map::place_items(const item_group_id&, int, const tripoint&, const tripoint&, bool, const time_point&, int, int)
 FILE     : src/mapgen.cpp
 LINE     : 5254
 VERSION  : ed2f630
```

### Steps To Reproduce

Find a hunting lodge

### Expected behavior

No errors

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 33 (Thirty Three); Release: 33; Codename: ThirtyThree; 
- Game Version: ed2f630 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

### Additional context

https://github.com/CleverRaven/Cataclysm-DDA/blob/ed2f630d83ecdf6033cb457ac9525759b7c91dc3/data/json/mapgen/nested/lodge_nested.json#L163

https://github.com/CleverRaven/Cataclysm-DDA/blob/ed2f630d83ecdf6033cb457ac9525759b7c91dc3/data/json/mapgen/nested/lodge_nested.json#L215-L225

**Comment by actual-nh** [comment]
Ping: @Maleclypse?

**Comment by jbytheway** [comment]
Fixed by #50684
